app/admin/event.rb

```diff
@@ -58,7 +54,7 @@ def set_users_before_update
       f.input :confirmed
 
       f.input :users,
-              collection: User.all.map { |user| [user.email, user.id] },
+              collection: User.all.map { |user| [user.first_name + ' ' + user.last_name, user.id] },
```

Comment:
user.first_name + ' ' + user.last_name is a string. better to use string with double quotes